src/network-config.js

```diff
@@ -87,6 +87,28 @@ export const networkConfigs = {
     },
     providers: [{ id: 'provided' }, { id: 'frame' }],
   },
+  xdai: {
+    addresses: {
+      ensRegistry:
+        localEnsRegistryAddress || '0xaafca6b0c89521752e559650206d7c925fd0e530',
+    },
+    nodes: {
+      defaultEth: 'wss://xdai.poanetwork.dev/wss',
+    },
+    settings: {
+      chainId: 100,
+      name: 'xDai',
+      shortName: 'xdai',
+      type: 'xdai',
+    },
+    // providers: ['injected', 'frame'],
+    providers: [
```

Comment:
Do Fortmatic and Portis support xDAI? AFAIK I think only Portis did?

Reply:
Woops, seems I didn't commit a few changes here to delete the above comment + the fortmatic provider. Will remove 👍